joint_state_publisher/joint_state_publisher/joint_state_publisher

```diff
@@ -49,15 +49,25 @@ class JointStatePublisher():
                 if jtype == 'continuous':
                     minval = -pi
                     maxval = pi
+                    try:
+                        limit = child.getElementsByTagName('limit')[0]
+                    	minVel = 0
+                        maxVel = float(limit.getAttribute('velocity'))
+		    except:
+                        rospy.logwarn("%s is not fixed, is continuous, but velocity limits are not specified!" % name)
```

Comment:
According to this wiki page:

http://wiki.ros.org/urdf/XML/joint

The `<limits>` tag is only required for `revolute` and `prismatic` joints, so it would be weird for this tool to complain about it not having it. I think it should fail more gracefully in this case and the warning message should be a suggestion not a declaration that something is wrong. For other cases, like `revolute`, this is captured in the following `else:` statement.